pyresample/test/test_utils.py

```diff
@@ -647,6 +647,313 @@ def test_norotation_yaml(self):
         os.remove(f.name)
 
 
+class TestLoadCFArea_Public(unittest.TestCase):
+    """ Test loading an area definition from netCDF/CF files using the
+        public API load_cf_area() """
+
+    def test_load_cf_from_wrong_filepath(self):
+        from pyresample.utils import load_cf_area
+
+        # wrong case #1: the path does not exist
+        cf_file = os.path.join(os.path.dirname(__file__), 'test_files', 'does_not_exist.nc')
+        self.assertRaises(FileNotFoundError, load_cf_area, cf_file)
+
+        # wrong case #2: the path exists, but is not a netCDF file
+        cf_file = os.path.join(os.path.dirname(__file__), 'test_files', 'areas.yaml')
+        self.assertRaises(ValueError, load_cf_area, cf_file)
+
+    def test_load_cf_parameters_errors(self):
+        from pyresample.utils import load_cf_area
+
+        cf_file = os.path.join(os.path.dirname(__file__), 'test_files', 'cf_nh10km.nc')
```

Comment:
I'm not in favour of having binary files added to the repo. I'd rather the nc file was created just for the test, or maybe better, that xarray's `open_dataset` would be mocked and the Dataset would be created here.

Reply:
Solved. The three netCDF/CF files are removed from test_files, and the tests are now run on xarray.Dataset objects.